HetDepot/Controllers/Visitor/ReservationForGroupController.cs

```diff
@@ -0,0 +1,58 @@
+﻿using HetDepot.People.Model;
+using HetDepot.Tours.Model;
+using HetDepot.Views;
+using HetDepot.Views.Parts;
+
+namespace HetDepot.Controllers;
+
+public class ReservationForGroupController : Controller
+{
+    private Tour _tour;
+    private Visitor _visitor; // TODO Add this visitor to the reservationAsGroupOwner
```

Comment:
Begrijp ik het goed dat een visitor als 'hoofd aanmelder' wordt gezien?

Reply:
Goede vraag, dat is denk ik even een keuze die we moeten maken. Ooit is er gesproken over het kunnen afmelden van de hele groep. Nu denk ik dat het handige feature is maar vraag me af hoe nodig.

Reply:
Even terug gekeken in FO en dit komt niet meer terug